Add How To Acknowledge Contributors file (#125)
r-devel/rdevguide: HOWTO-acknowledge-contributors.md

To recognise the contributions made on this repository we use the [All Contributors](https://allcontributors.org/) Bot. 
Our aim is to recognise all contributors including those that don't push code. 

> Some (but not limited to) possible contributions which the bot supports include:

Reviewer: ```suggestion
Types of contribution supported by the bot include:
```

- Creating a new Pull Request or Issue on this repository
- Giving talks based on this repository
- Mentoring contributions to this repository
- Reviewing Pull Requests on this repository
- Reporting or working on accessibility issues related to this repository
- Providing links to podcasts, interviews, or any audio material, blogposts, tutorials, talks, videos, slides, repository, and recordings that mention this repository or mention the R Development Guide.
These links can be shared through a pull request or by opening a new issue.
- Adding content to the Wiki on this repository
- Creating logo/iconography/visual design/etc. for this project
- Organising events related to this project
- Providing financial support for this project 
- Support the infrastructure required for this project
- Help with the maintenance of this repository
- Promoting this repository and project
- Answering Questions in Issues, Slack, etc.
- Translating content of this repository
- Conducting literature review for the work on this project

> 

Reviewer: Can you add a link at the end of where to look for a full list (although this is already very comprehensive!).

# How to add Contributor(s)?

1. To add a Contributor to this repository, comment on the relevant Issue or Pull Request, asking `@all-contributors` to add a Contributor:

   ```sh
   @all-contributors please add @<github-username> for <contribution>
   ```

> 2. After this the bot will The bot create a Pull Request to add the Contributor. Further it reply with the pull request details. 

Reviewer: ```suggestion
2. After this the bot will create a Pull Request to add the Contributor and reply to your comment with the pull request details. 
```
